osu.Game.Rulesets.Osu/Difficulty/Evaluators/AimEvaluator.cs

```diff
@@ -112,18 +118,50 @@ public static double EvaluateDifficultyOf(DifficultyHitObject current, bool with
                 velocityChangeBonus *= Math.Pow(Math.Min(osuCurrObj.StrainTime, osuLastObj.StrainTime) / Math.Max(osuCurrObj.StrainTime, osuLastObj.StrainTime), 2);
             }
 
-            if (osuLastObj.BaseObject is Slider)
+            // Add in acute angle bonus or wide angle bonus + velocity change bonus, whichever is larger.
+            aimStrain += Math.Max(acuteAngleBonus * acute_angle_multiplier, wideAngleBonus * wide_angle_multiplier + velocityChangeBonus * velocity_change_multiplier);
+
+            double sliderJumpBonus = 0;
             {
-                // Reward sliders based on velocity.
-                sliderBonus = osuLastObj.TravelDistance / osuLastObj.TravelTime;
+                if (osuLastObj.BaseObject is Slider slider)
+                {
+                    // Take just slider heads into account because we're computing sliderjumps, not slideraim
+                    sliderJumpBonus = slider_jump_multiplier * aimStrain * sliderJumpsAdjustingMultiplier;
+
+                    // Reward more if sliders and circles are alternating (actually it's still lower than several sliders in a row)
```

Comment:
What is the rationale behind this? If we're alternating between sliders and circles then you're saying it's more difficult to aim than being slider jumps only.. but why?

Reply:
in general - the meaning of buffing slider jumps is accounting for the fact that slider jumps are more difficult to process/read in comparison to normal jumps
but the cases where there's some notes in the middle of slider jump section or vice versa - it's not becoming any easier to read than just all sliders

Reply:
you can create jumps test map with three cases:
- only circles
- alternating
- only sliders
in my experience last two have similar difficulty, but maybe i'm just skill-issued

Reply:
I just tested the jumps here:
With alternating bonus:
- normal - 3.38*
- alternating - 3.48*
- sliders - 3.58

Without:
- normal - 3.38*
- alternating - 3.45*
- sliders - 3.58

Reply:
maybe i actually should revert alternating multiplier to 1.0x

Reply:
even with 2.0x it's still only 3.55*